### ISSUE TITLE
Image tag in description not working

### ISSUE THREAD

**Issue by skellitor301** [FactorioBlueprints/factorio-prints#1022]
In the github markdown doc, there is a section to include images as an embedded object. However, when I try to use the markdown `![alt text](img url)` or use html to embed an image, the broken image icon appears in place of the image and the alt text is shown right next to it. It would be beneficial to include image embeds in the description to allow things like steps to use to include screenshots.\
Here's a good example of this being used

---

#### Setup
- You'll need to set up a space between your storage network and your crafters, requesters, cargo bay where you can put a line of these. In the example image below, I've set up my crafters going right and my storage going left.\
![setting up space](https://imgur.com/em82YeX.png)
- Next, place the blueprint with the arrows pointing towards he crafters.\
![placing blueprint](https://i.imgur.com/PdSOeqg.png)

**Comment by FactorioBlueprints** [comment]
Are the embedded images in imgur?

**Comment by skellitor301** [comment]
Yes, I've even tried changing the url to match what is required for the thumbnail. But no matter what I do it results in a broken image link

**Comment by motlin** [comment]
I think it's related to imgur specifically. Images elsewhere embed without issue.

**Comment by skellitor301** [comment]
If that were the case then they would not embed here. But I manage to get the image tag to work on Github, while on the site the same tag doesn't work.

**Comment by FactorioBlueprints** [comment]
Yes I see the behavior that you're describing but I'm not sure what the fix is or if it's possible to fix. Imgur links embedded elsewhere seem to work. Images to elsewhere seem to work when embedded in factorioprints. Other embeds to imgur also work from factorioprints, when including the noreferrer wrapper. Maybe it would be possible to configure the markdown renderer to create noreferrer wrappers. I could use help with this one.

**Comment by skellitor301** [comment]
I wish I could help, but I don't have the coding experience to give a definitive answer. Maybe the community board on Github could provide insight?
